python/observability-online-eval/coding-agent/.env.example

```
@@ -0,0 +1,3 @@
+OPENAI_API_KEY=your_openai_api_key
+MAXIM_API_KEY=your_maxim_api_key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MAXIM_API_KEY key should go before the OPENAI_API_KEY key

(UnorderedKey)